aiperf/post_processors/record_export_results_processor.py

```
@@ -0,0 +1,88 @@
+
+                async with aiofiles.open(
+                    self.output_file, mode="a", encoding="utf-8"
+                ) as f:
+                    await f.write(json_str)
+                    await f.write("\n")
+
+                self.record_count += 1
+                if self.record_count % 100 == 0:
+                    self.debug(f"Wrote {self.record_count} record metrics")
+
+            except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)